app/src/main/java/org/schabi/newpipe/fragments/subscription/SubscriptionFragment.java

```diff
@@ -103,9 +131,110 @@ public void onDestroy() {
         super.onDestroy();
     }
 
-    ///////////////////////////////////////////////////////////////////////////
+    /*/////////////////////////////////////////////////////////////////////////
+    // Menu
+    /////////////////////////////////////////////////////////////////////////*/
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+
+        ActionBar supportActionBar = activity.getSupportActionBar();
+        if (supportActionBar != null) {
+            supportActionBar.setDisplayShowTitleEnabled(true);
+            setTitle(getString(R.string.tab_subscriptions));
+        }
+
+        inflater.inflate(R.menu.menu_subscriptions, menu);
+        setupImportSubMenu(menu);
+    }
+
+    private void setupImportSubMenu(final Menu menu) {
```

Comment:
Just a thought, but wouldn't be better if we move the import/export menu into the fragment as part of the list header (along with `What's New`)? Right now, this feature might not be easily discoverable, as users are probably used to the menu staying the same across all pagers.